Hackney.Shared.Tenure/Sns/User.cs

```diff
@@ -0,0 +1,8 @@
+namespace Hackney.Shared.Tenure.Sns
+{
+    public class User
+    {
+        public string Name { get; set; }
+        public string Email { get; set; }
+    }
+}
```

Comment:
Not needed here. The Sns event object is common but should not go in this package.